pkg/scheduler/framework/plugins/nodeaffinity/node_affinity.go

```diff
@@ -192,6 +192,10 @@ func (pl *NodeAffinity) PreScore(ctx context.Context, cycleState *framework.Cycl
 	if err != nil {
 		return framework.AsStatus(err)
 	}
```

Comment:
ideally, we'd better have a testcase to cover this code branch, i.e. the `PreScore` is enabled, but expected an `err` status.

Reply:
done